exercises/java/encryption-context-complete/pom.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>sfw.example.esdkworkshop</groupId>
+    <artifactId>esdkworkshop</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>esdkworkshop</name>
+    <description>A simple esdkworkshop.</description>
+    <!-- FIXME change it to the project's website -->
+    <url>http://www.example.com</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+    <dependencies>
```

Comment:
Do you want to add ACCP to this list?

https://aws.amazon.com/blogs/opensource/introducing-amazon-corretto-crypto-provider-accp/

Reply:
D'oh! I knew I was forgetting something. :) Absolutely!